config/config.txt

```diff
@@ -549,6 +549,20 @@ RESPECT_GLOBAL_BANS
 MAX_SHUTTLE_COUNT 6
 MAX_SHUTTLE_SIZE 300
 
+### Shuttle ruin save settings
+## Uncomment to prevent saving shuttle dmm files to the server.
+SAVE_SHUTTLE_RUINS
+## Location where shuttle ruins are saved to. Must end in /
+SHUTTLE_RUIN_FILEPATH data/customShuttleSaves/
+## Max filesize in kB that shuttles are allowed to take up on server
+## Default: 1 megabyte
+SHUTTLE_TOTAL_FILESIZE_MAX 1000
+## Max filesize a single shuttle is allowed to take up
+## Default: 40 kB (40,000 characters)
+SHUTTLE_FILESIZE_MAX 40
+## Max amount of shuttles that can spawn at roundstart by SSorbits
+ROUNDSTART_ABANDONED_RUINS 5
+
```

Comment:
(This was never actually removed from config, so config changes don't need to be applied.)